CoverMyDotNet/Requests/GetRequestPage.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Text;
+using RestSharp;
+
+namespace CoverMyDotNet.Requests
+{
+	public class GetRequestPage : RestRequest
+	{
+		public GetRequestPage(string apiId, string requestId, string tokenId, string resource = "")
+		{
+			this.Method = Method.GET;
+			this.Resource = string.IsNullOrEmpty(resource) ? 
+								string.Format("request-pages/{0}", requestId) : resource;
+			this.RootElement = "request_page";
+			this.AddQueryParameter("v", "1");
+			this.AddParameter("Accept", "application/typed+json", ParameterType.HttpHeader);
+			this.AddParameter("Authorization", string.Format("Bearer {0}+{1}", apiId, tokenId), ParameterType.HttpHeader);			
```

Comment:
The auth seems to vary on different endpoints. Only some take bearer with api key and token, others take just bearer with just the api key, and some take basic auth. I am not sure how useful making a base class would be if we were just going to be using it for a few cases.